### PR TITLE
Fix deprecation warnings introduced in the upgrade to 0.13.13.

### PR DESCRIPTION
**Pull request by jkinkead**
I figured out how to fix the two last warnings, AND figured out the magic concise `dependsOn` invocation. Hooray!

### CODE REVIEW

**Comment by jkinkead** [comment]
Oh, it's worth noting that this also has a couple of formatting fixes.

File: src/main/scala/org/allenai/plugins/StylePlugin.scala [hunk 1]
```diff
@@ -100,22 +102,20 @@ class SbtLogOutput[T <: FileSpec](logger: Logger) extends ScalastyleOutput[T] {
...
   }
 
-  private[this]
-  def location(file: T, line: Option[Int], column: Option[Int]): String =
+  private[this] def location(file: T, line: Option[Int], column: Option[Int]): String =
     (file.name +
-     line.map(n => ":" + n + column.map(":" + _).getOrElse(""))
-         .getOrElse(""))
+      line.map(n => ":" + n + column.map(":" + _).getOrElse(""))
+      .getOrElse(""))
```
Comment (ColinArenz) [inline src/main/scala/org/allenai/plugins/StylePlugin.scala]: Nit-pick: Slightly weird formatting here is hard to follow. Maybe a block with some intermediate variables?

Comment (jkinkead) [inline src/main/scala/org/allenai/plugins/StylePlugin.scala]: IIRC much of this plugin was copy-pasted from another sbt plugin's implementation.

I made this use a procedural `StringBuilder` because I am a REBEL.

Comment (ColinArenz) [inline src/main/scala/org/allenai/plugins/StylePlugin.scala]: Living dangerously. I like the new version much better.
